### ISSUE TITLE
GERS uses but doesn't define certain ProductNotices (SKC and DEL)

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2225]
_Imported from trac ticket 2225.
Created by **kbower**
Opened in trac: Mar 14, 2013, 1:08:02 PM
Last modified in trac: Mar 14, 2013, 3:30:48 PM_
***
We need to add these or they cause problems after [r3904](../commit/5bd7dc4eb44fabafdab97e55b65b6210ce99b8c8), see #2220.

```
SQL> select distinct obsoleteid from products where obsoleteid is not null and not exists (select 0 from obsoletes where obsoletes.obsoleteid = products.obsoleteid);

OBS
---
DEL
SKC
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 14, 2013, 1:13:44 PM:_

Your system has products with DROP_CD "DEL" and/or "SKC". Apparently "SKC" means GERS sku merge utility was used to merge/change sku.

What does "DEL" mean?

Thanks!
Kent

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 14, 2013, 2:27:49 PM:_

Thanks Doug, I think you are probably correct... I'm just going to make Loft act like PREVENT_PO = 'Y' and PREVENT_SALE = 'Y' for invalid DROP_CDs.

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 14, 2013, 3:30:48 PM:_

(Changeset [[3907]](../commit/22ec1834da31da0d90654e2232c530fd49156773))
* add 'SKC' ProductNotice (Drop_CD) since GERS doesn't add it (see #2225)
 * for undefined productnotices, including "DEL", treat as if "Blocks sale" and note that it is invalid (fixes #2225)
 * fix loading ProductNotice for valueproduct in "Related Items" section
 * fix warnings to appear if adding via the "Related Items" section

**Comment by kentbower** [comment]
_**kbower** trac status change on Mar 14, 2013, 3:30:48 PM:_ **closed**
